include/OpenKit/AbstractOpenKitBuilder.h

```diff
@@ -53,8 +54,17 @@ namespace openkit
 			/// If a custom logger is provided (by calling @ref withLogger(std::shared_ptr<openkit::ILogger>)) debug and info log output
 			/// depends on the values returned by @ref openkit::ILogger::isDebugEnabled() and @ref openkit::ILogger::isInfoEnabled().
  			/// @return @c this for fluent usage
+			/// @deprecated Use @ref AbstractOpenKitBuilder::withLogLevel instead.
```

Comment:
The "AbstractOpenKitBuilder::" is probably unneeded. In the description to "withLogLevel" you mention "@ref withLogger" without "AbstractOpenKitBuilder::". Both should be the same